test/math/ec_scalar.cpp

```diff
@@ -50,6 +50,24 @@ BOOST_AUTO_TEST_CASE(ec_scalar__add__valid__expected)
     BOOST_REQUIRE_EQUAL(encode_base16(sum.secret()), SUM1);
 }
 
+BOOST_AUTO_TEST_CASE(ec_scalar__from__uint64__valid__expected)
+{
+    const ec_scalar x(110);
+    const ec_scalar y(256);
+    const auto sum = x + y;
+    BOOST_REQUIRE(sum);
+    const ec_scalar result(110 + 256);
```

Comment:
All things being equal I recommend use of “expected” as the expectation var.

Reply:
Done